pygments/styles/nord.py

```diff
@@ -0,0 +1,160 @@
+"""
+    pygments.styles.nord
+    ~~~~~~~~~~~~~~~~~~~~~~~
+
+    pygments version of the "nord" theme by Arctic Ice Studio
+    https://www.nordtheme.com/
+
+    :copyright: Copyright 2006-2021 by the Pygments team, see AUTHORS.
+    :license: BSD-3 Clause License. See LICENSE for details
+"""
+
+from pygments.style import Style
+from pygments.token import (
+    Keyword,
+    Name,
+    Comment,
+    String,
+    Error,
+    Number,
+    Operator,
+    Generic,
+    Whitespace,
+    Punctuation,
+    Text,
+)
+
+
+class NordStyle(Style):
+    """
+    Pygments version of the "nord" theme by Arctic Ice Studio
+    """
+
+    linenos = "#FF0000"
```

Comment:
I don't think it does anything. It might be leftover of a few tests I was making.

The thing is, I don't know how to make the line_number get the correct color. Is it something in my configuration?

```bash
pygmentize -v -o random_number_1_10.png -O hl_lines="1 3 4 ",style=nord-darker,line_numbers=True,font_name=FiraCode -f png random_number_1_10.sh
```

The above command generates the image below.

![random_number_1_10](https://user-images.githubusercontent.com/2037375/179212783-e06c8756-f690-4b89-b544-d10893941ae3.png)

Reply:
Even if these ridiculous colors, it doesn't affect the image:

```python
    line_number_color = "#FF0000"
    line_number_background_color = "#00FF00"
    line_number_special_color = "#0000FF"
    line_number_special_background_color = "#0000FF"
```

Reply:
Do you know how to get the color from line number to change?